packages/fake-driver/lib/driver.js

```diff
@@ -205,82 +232,354 @@ export class FakeDriver extends BaseDriver {
   /*********
    * ALERT *
    *********/
-  assertNoAlert = alert.assertNoAlert;
-  assertAlert = alert.assertAlert;
-  getAlertText = alert.getAlertText;
-  setAlertText = alert.setAlertText;
-  postAcceptAlert = alert.postAcceptAlert;
-  postDismissAlert = alert.postDismissAlert;
+  /**
```

Comment:
ugh i hate that one line is now 6. it makes it much less desirable to use this file-splitting pattern

Reply:
I kind of hate it too.  there's another way, which is to just fake it.  let me see if I can do that.